_inc/client/plans/single-product-search/index.jsx

```diff
@@ -93,6 +94,11 @@ export function SingleProductSearchCard( props ) {
 						'Your current site record size: %s records',
 						{ args: numberFormat( recordCount ), count: recordCount }
 					) }
+					<InfoPopover position="right">
+						{
+							'Records are all posts, pages, custom post types and other types of content indexed by Jetpack Search.'
```

Comment:
This needs to use the oxford comma and be wrapped in a translation function call:
```suggestion
							__( 'Records are all posts, pages, custom post types, and other types of content indexed by Jetpack Search.' )
```